packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -318,6 +342,38 @@ class Auth {
         )
     }
 
+    /**
+     * This is a wrapper method for ShopperCustomer API registerCustomer endpoint.
+     *
+     */
+    async register(
+        body: Argument<ShopperCustomers<ApiClientConfigParams>['registerCustomer']>['body']
+    ) {
+        const {
+            customer: {email},
+            password
+        } = body
+        const isGuest = false
+
+        // email is optional field from isomorphic library
+        // type CustomerRegistration
+        // here we had to guard it to avoid ts error
+        if (!email) {
+            throw new Error('Missing email')
```

Comment:
```suggestion
            throw new Error('Customer registration is missing email address.')
```
More specific about _what_ is missing the email. 🤷